### PR TITLE
ROE-1811 Update text for presenter page

### PR DESCRIPTION
**Pull request by mattch1**
### JIRA link

https://companieshouse.atlassian.net/browse/ROE-1811

### Change description

Add hint text to the top of the page.
Update public register text.

### Work checklist

- [x] Tests added where applicable
- [x] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: views/presenter.html [hunk 1]
```diff
@@ -44,8 +45,8 @@ <h1 class="govuk-heading-xl">Who can we contact about this application?</h1>
 
         {{ govukInsetText({
           html: '
-            <h2 class="govuk-heading-m">Information shown on the public register</h2>
-            <p>None of the information provided in this section will be shown on the public Register of Overseas Entities.</p>
+            <h2 class="govuk-heading-m">What information we’ll show on the public register</h2>
```
Comment (mouhajer-ch) [inline views/presenter.html:48]: Why you do not want to use the `inset-text` as [here](https://github.com/companieshouse/overseas-entities-web/blob/main/views/includes/inset-text.html)?

Comment (mattch1) [reply views/presenter.html:48]: Inset text will add a bar to the left hand side, which is not what has been requested. Also this is more consistent with top page hint text for other pages such as due diligence.

Comment (mouhajer-ch) [reply views/presenter.html:48]: Could you please check the screenshot on your story. Anyway they should be all in the same style, so eventually check with UX and confirm that all the inset text related to the information shown on the public register should have all the same style.

Comment (mattch1) [reply views/presenter.html:48]: Sorry thought you were talking about something else , yes we can use inset text here.

Comment (mattch1) [reply views/presenter.html]: Updated this.

**Review by mouhajer-ch — APPROVED** [review]

**Pull request merged**